pipelines/HSC/DRP-RC2.yaml

```diff
@@ -89,10 +89,23 @@ tasks:
   forcedPhotCcd:
     class: lsst.meas.base.forcedPhotCcd.ForcedPhotCcdTask
     config:
-      # The application of external calibrations doesn't work with Gen3 now (DM-17062)
-      doApplyExternalPhotoCalib: false
-      doApplyExternalSkyWcs: false
-      doApplySkyCorr: false
+      doApplyExternalPhotoCalib: true
+      doApplyExternalSkyWcs: true
+  forcedPhotDiffim:
+    class: lsst.meas.base.ForcedPhotCcdTask
+    config:
+      doApplyExternalPhotoCalib: true
+      doApplyExternalSkyWcs: true
+  forcedPhotCcdOnDiaObjects:
+    class: lsst.meas.base.ForcedPhotCcdFromDataFrameTask
```

Comment:
Delete these overrides and friends below, and make an `obs_subaru/config/forcedPhotCcdFromDataFrame.py`. 

Applying jointcal/FGCM during diffim  is still a TODO, so making these true is kinda inconsistent (if there'a 1-observation diaObject it won't be force photometered at the  exact xy of its centroid as measured on that one diffim).  I could go either way, since we already know that what we have isn't the best that we do anyway, and will be fixed in April. You're welcome to make a note to remind me to do this on  DM-31292 if you don't want to make a `obs_subaru/config/forcedPhotCcdFromDataFrame.py` now.    

But either way, these overrides here go away.

Reply:
I don't exactly follow your logic, I think we should discuss later.  In the meantime I'll comment on DM-31292.